system_test/common/aest_compatibility_SUITE_data/aeternity.yaml.mustache

```diff
@@ -19,7 +19,7 @@ keys:
     dir: ./keys
 
 chain:
-    persist: true
+    persist: false
```

Comment:
Eventually I did not rely on this though I suggest to keep disabled.

Reply:
Isn't this going to consume more memory then ? Eventually slow down the execution 🤔

Reply:
It shall not matter - no txs and short chain - though reverted in 15f6e8b0.

Reply:
It shall not matter - no txs and short chain - though reverted in 15f6e8b0.